tests/components/modbus/conftest.py

```diff
@@ -67,6 +67,12 @@ def config_addon():
     return None
 
 
+@pytest.fixture
+def do_exception():
+    """Remove side_effect to pymodbus calls."""
+    return False
```

Comment:
This is ok now since we only need one kind of exception. If we need more flexibility in the future we can change this to return the actual side effect / exception. Then we can overwrite this fixture to return whatever side effect we need.